### PR TITLE
Updated Shortcut Legend 

### PR DESCRIPTION
**Pull request by AkalUstat**
- fixes Win10 (added menu option for Win10)
- updated the legend to match new changes in #649

### CODE REVIEW

File: LEGEND.md [hunk 1]
```diff
@@ -10,54 +10,39 @@
...
 
 ### <a name="slides">Slide Shortcuts</a>
 
-#### MacOS:
+#### macOS:
 
-Cmd + e = empty slide
-Cmd + 1 = Waheguru Slide
+- Cmd + 1 = Waheguru Slide
+- Cmd + 2 = Ik Oankar Slide
```
Comment (maneetpaul) [inline LEGEND.md]: We should rename this "Mool Mantra"

File: LEGEND.md [hunk 2]
```diff
@@ -10,54 +10,39 @@
...
+- Cmd + 3 = Empty Slide
+- Cmd + 4 = Anand Sahib (Bhog)
+- Cmd + 5 = Help Guide
+- Cmd + 6 = Shortcut Legend
```
Comment (maneetpaul) [inline LEGEND.md]: Do we really need a shortcut for the help/shortcut guides? I think we can omit these.

File: LEGEND.md [hunk 3]
```diff
@@ -10,54 +10,39 @@
...
-
-Ctrl + z = undo
-
-Ctrl + x = cut
-
-Ctrl + c = copy
-
-Ctrl + v = paste
+- Ctrl + q = Close app (shut down app)
+- _Some Windows Shortcuts are currently broken and will be fixed in a later update_
```
Comment (maneetpaul) [inline LEGEND.md]: Which ones are broken?

File: LEGEND.md (hunk 2 above)
Comment (AkalUstat) [reply LEGEND.md]: ok

File: LEGEND.md (hunk 3 above)
Comment (AkalUstat) [reply LEGEND.md]: #637

File: LEGEND.md (hunk 1 above)
Comment (AkalUstat) [reply LEGEND.md]: 👍

**Pull request merged**